.github/workflows/pypitest.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         environment-file: environment.yml
         environment-name: smmregrid
         cache-downloads: true
-        cache-environment: true
+        cache-environment: false
```

Comment:
I'd revert this as well before merging

Reply:
well, it depends if we keep or not the python matrix. I have noticed test failure when retrieving the cached env with matrix active, not fully sure if these are related though.